*** Configuring hosts
h1 h2 r1 
[sidecar] Setting congestion control to cubic
[sidecar] tso and gso are OFF
[sidecar] Starting the NGINX/Python webserver on h1...
[sidecar] NOT starting the TCP PEP or sidecar
*** h2 : ('python3 mininet/client.py -n 200k --http 3 --stdout /dev/null --stderr /home/gina/sidecar/error.log -cc cubic --loss 2 --log-level error -t 10 ',)
head -c 200k /dev/urandom > /tmp/tmp50g7rxwg
Data Size: 200k
HTTP: --http3
RUST_LOG=error sidecurl --http3 --quiche-cc cubic --data-binary @/tmp/tmp50g7rxwg --insecure https://10.0.1.10:443/ -w "%{time_connect}\t%{time_appconnect}\t%{time_starttransfer}\t\t%{time_total}\t%{exitcode}\t\t%{response_code}\t\t%{size_upload}\t\t%{size_download}\t%{errormsg}\n" --max-time 15 -o /dev/null 2>>/home/gina/sidecar/error.log 
time_connect	time_appconnect	time_starttransfer	time_total	exitcode	response_code	size_upload	size_download	errormsg
0.318880	0.000000	0.318939		2.460089	0		200		204800		18	
0.158266	0.000000	0.158322		1.710818	0		200		204800		18	
0.159321	0.000000	0.159380		14.999013	28		000		204800		0	Operation timed out after 14999 milliseconds with 0 bytes received
0.158366	0.000000	0.158423		1.094456	0		200		204800		18	
0.158376	0.000000	0.158432		0.968442	0		200		204800		18	
0.158519	0.000000	0.158575		1.584787	0		200		204800		18	
0.158605	0.000000	0.158661		1.579999	0		200		204800		18	
0.158326	0.000000	0.158382		1.819824	0		200		204800		18	
0.158280	0.000000	0.158335		14.999014	28		000		14521		0	Operation timed out after 14999 milliseconds with 0 bytes received
0.158372	0.000000	0.158430		2.148661	0		200		204800		18	
*** Stopping 0 controllers

*** Stopping 2 links
..
*** Stopping 0 switches

*** Stopping 3 hosts
h1 h2 r1 
*** Done
max_queue_size (bytes) = 190000.0
*** Configuring hosts
h1 h2 r1 
[sidecar] Setting congestion control to cubic
[sidecar] tso and gso are OFF
[sidecar] Starting the NGINX/Python webserver on h1...
[sidecar] NOT starting the TCP PEP or sidecar
*** h2 : ('python3 mininet/client.py -n 400k --http 3 --stdout /dev/null --stderr /home/gina/sidecar/error.log -cc cubic --loss 2 --log-level error -t 10 ',)
head -c 400k /dev/urandom > /tmp/tmprs6lhfkk
Data Size: 400k
HTTP: --http3
RUST_LOG=error sidecurl --http3 --quiche-cc cubic --data-binary @/tmp/tmprs6lhfkk --insecure https://10.0.1.10:443/ -w "%{time_connect}\t%{time_appconnect}\t%{time_starttransfer}\t\t%{time_total}\t%{exitcode}\t\t%{response_code}\t\t%{size_upload}\t\t%{size_download}\t%{errormsg}\n" --max-time 15 -o /dev/null 2>>/home/gina/sidecar/error.log 
time_connect	time_appconnect	time_starttransfer	time_total	exitcode	response_code	size_upload	size_download	errormsg
0.326309	0.000000	0.326367		4.138527	0		200		409600		18	
0.164019	0.000000	0.164076		2.063709	0		200		409600		18	
0.158307	0.000000	0.158361		3.964731	0		200		409600		18	
0.158273	0.000000	0.158325		1.311373	0		200		409600		18	
0.540836	0.000000	0.540892		2.092792	0		200		409600		18	
0.158241	0.000000	0.158296		2.313918	0		200		409600		18	
0.158303	0.000000	0.158360		1.701279	0		200		409600		18	
1.463495	0.000000	1.463552		3.610508	0		200		409600		18	
0.158458	0.000000	0.158512		2.039053	0		200		409600		18	
0.158330	0.000000	0.158389		3.357007	0		200		409600		18	
*** Stopping 0 controllers

*** Stopping 2 links
..
*** Stopping 0 switches

*** Stopping 3 hosts
h1 h2 r1 
*** Done
max_queue_size (bytes) = 190000.0
*** Configuring hosts
h1 h2 r1 
[sidecar] Setting congestion control to cubic
[sidecar] tso and gso are OFF
[sidecar] Starting the NGINX/Python webserver on h1...
[sidecar] NOT starting the TCP PEP or sidecar
*** h2 : ('python3 mininet/client.py -n 600k --http 3 --stdout /dev/null --stderr /home/gina/sidecar/error.log -cc cubic --loss 2 --log-level error -t 10 ',)
head -c 600k /dev/urandom > /tmp/tmpfsuz1dwg
Data Size: 600k
HTTP: --http3
RUST_LOG=error sidecurl --http3 --quiche-cc cubic --data-binary @/tmp/tmpfsuz1dwg --insecure https://10.0.1.10:443/ -w "%{time_connect}\t%{time_appconnect}\t%{time_starttransfer}\t\t%{time_total}\t%{exitcode}\t\t%{response_code}\t\t%{size_upload}\t\t%{size_download}\t%{errormsg}\n" --max-time 15 -o /dev/null 2>>/home/gina/sidecar/error.log 
time_connect	time_appconnect	time_starttransfer	time_total	exitcode	response_code	size_upload	size_download	errormsg
0.318837	0.000000	0.318894		2.208504	0		200		614400		18	
0.158247	0.000000	0.158301		2.273486	0		200		614400		18	
0.158361	0.000000	0.158417		2.772529	0		200		614400		18	
0.158508	0.000000	0.158566		2.733828	0		200		614400		18	
0.158318	0.000000	0.158375		2.609174	0		200		614400		18	
0.158313	0.000000	0.158369		2.641374	0		200		614400		18	
0.158229	0.000000	0.158286		2.421101	0		200		614400		18	
0.158286	0.000000	0.158341		5.798675	0		200		614400		18	
0.158316	0.000000	0.158372		14.999013	28		000		14521		0	Operation timed out after 14999 milliseconds with 0 bytes received
1.464564	0.000000	1.464621		3.496341	0		200		614400		18	
*** Stopping 0 controllers

*** Stopping 2 links
..
*** Stopping 0 switches

*** Stopping 3 hosts
h1 h2 r1 
*** Done
max_queue_size (bytes) = 190000.0
*** Configuring hosts
h1 h2 r1 
[sidecar] Setting congestion control to cubic
[sidecar] tso and gso are OFF
[sidecar] Starting the NGINX/Python webserver on h1...
[sidecar] NOT starting the TCP PEP or sidecar
*** h2 : ('python3 mininet/client.py -n 800k --http 3 --stdout /dev/null --stderr /home/gina/sidecar/error.log -cc cubic --loss 2 --log-level error -t 10 ',)
head -c 800k /dev/urandom > /tmp/tmpcw1ikzm4
Data Size: 800k
HTTP: --http3
RUST_LOG=error sidecurl --http3 --quiche-cc cubic --data-binary @/tmp/tmpcw1ikzm4 --insecure https://10.0.1.10:443/ -w "%{time_connect}\t%{time_appconnect}\t%{time_starttransfer}\t\t%{time_total}\t%{exitcode}\t\t%{response_code}\t\t%{size_upload}\t\t%{size_download}\t%{errormsg}\n" --max-time 15 -o /dev/null 2>>/home/gina/sidecar/error.log 
time_connect	time_appconnect	time_starttransfer	time_total	exitcode	response_code	size_upload	size_download	errormsg
0.318902	0.000000	0.318959		2.804296	0		200		819200		18	
0.158223	0.000000	0.158279		6.428207	0		200		819200		18	
0.158324	0.000000	0.158381		2.629353	0		200		819200		18	
0.158299	0.000000	0.158359		4.887612	0		200		819200		18	
0.158302	0.000000	0.158361		2.194586	0		200		819200		18	
0.158285	0.000000	0.158342		3.402119	0		200		819200		18	
0.158350	0.000000	0.158405		5.497638	0		200		819200		18	
0.158249	0.000000	0.158304		2.511462	0		200		819200		18	
0.158249	0.000000	0.158306		6.581850	0		200		819200		18	
0.163998	0.000000	0.164053		4.600964	0		200		819200		18	
*** Stopping 0 controllers

*** Stopping 2 links
..
*** Stopping 0 switches

*** Stopping 3 hosts
h1 h2 r1 
*** Done
max_queue_size (bytes) = 190000.0
*** Configuring hosts
h1 h2 r1 
[sidecar] Setting congestion control to cubic
[sidecar] tso and gso are OFF
[sidecar] Starting the NGINX/Python webserver on h1...
[sidecar] NOT starting the TCP PEP or sidecar
*** h2 : ('python3 mininet/client.py -n 1000k --http 3 --stdout /dev/null --stderr /home/gina/sidecar/error.log -cc cubic --loss 2 --log-level error -t 10 ',)
head -c 1000k /dev/urandom > /tmp/tmpz_kfmu2r
Data Size: 1000k
HTTP: --http3
RUST_LOG=error sidecurl --http3 --quiche-cc cubic --data-binary @/tmp/tmpz_kfmu2r --insecure https://10.0.1.10:443/ -w "%{time_connect}\t%{time_appconnect}\t%{time_starttransfer}\t\t%{time_total}\t%{exitcode}\t\t%{response_code}\t\t%{size_upload}\t\t%{size_download}\t%{errormsg}\n" --max-time 15 -o /dev/null 2>>/home/gina/sidecar/error.log 
time_connect	time_appconnect	time_starttransfer	time_total	exitcode	response_code	size_upload	size_download	errormsg
0.326178	0.000000	0.326239		5.829529	0		200		1024000		18	
0.158293	0.000000	0.158347		3.252448	0		200		1024000		18	
0.158231	0.000000	0.158289		3.139070	0		200		1024000		18	
0.158480	0.000000	0.158536		14.999015	28		000		1024000		0	Operation timed out after 14999 milliseconds with 0 bytes received
0.158377	0.000000	0.158434		6.268121	0		200		1024000		18	
0.158260	0.000000	0.158319		3.070533	0		200		1024000		18	
0.158259	0.000000	0.158313		7.188801	0		200		1024000		18	
0.158296	0.000000	0.158355		8.996511	0		200		1024000		18	
0.158496	0.000000	0.158553		6.588768	0		200		1024000		18	
0.158300	0.000000	0.158357		5.658664	0		200		1024000		18	
*** Stopping 0 controllers

*** Stopping 2 links
..
*** Stopping 0 switches

*** Stopping 3 hosts
h1 h2 r1 
*** Done
max_queue_size (bytes) = 190000.0
*** Configuring hosts
h1 h2 r1 
[sidecar] Setting congestion control to cubic
[sidecar] tso and gso are OFF
[sidecar] Starting the NGINX/Python webserver on h1...
[sidecar] NOT starting the TCP PEP or sidecar
*** h2 : ('python3 mininet/client.py -n 2000k --http 3 --stdout /dev/null --stderr /home/gina/sidecar/error.log -cc cubic --loss 2 --log-level error -t 10 ',)
head -c 2000k /dev/urandom > /tmp/tmpjcl3id0d
Data Size: 2000k
HTTP: --http3
RUST_LOG=error sidecurl --http3 --quiche-cc cubic --data-binary @/tmp/tmpjcl3id0d --insecure https://10.0.1.10:443/ -w "%{time_connect}\t%{time_appconnect}\t%{time_starttransfer}\t\t%{time_total}\t%{exitcode}\t\t%{response_code}\t\t%{size_upload}\t\t%{size_download}\t%{errormsg}\n" --max-time 20 -o /dev/null 2>>/home/gina/sidecar/error.log 
time_connect	time_appconnect	time_starttransfer	time_total	exitcode	response_code	size_upload	size_download	errormsg
0.319129	0.000000	0.319185		8.133054	0		200		2048000		18	
0.163937	0.000000	0.163997		9.693753	0		200		2048000		18	
0.163936	0.000000	0.163991		7.817766	0		200		2048000		18	
0.158240	0.000000	0.158297		5.541512	0		200		2048000		18	
0.158223	0.000000	0.158277		8.646523	0		200		2048000		18	
0.163913	0.000000	0.163967		5.441556	0		200		2048000		18	
0.163918	0.000000	0.163969		7.813523	0		200		2048000		18	
0.163943	0.000000	0.164000		9.046015	0		200		2048000		18	
0.164004	0.000000	0.164061		14.191820	0		200		2048000		18	
0.164060	0.000000	0.164120		9.011564	0		200		2048000		18	
*** Stopping 0 controllers

*** Stopping 2 links
..
*** Stopping 0 switches

*** Stopping 3 hosts
h1 h2 r1 
*** Done
max_queue_size (bytes) = 190000.0
*** Configuring hosts
h1 h2 r1 
[sidecar] Setting congestion control to cubic
[sidecar] tso and gso are OFF
[sidecar] Starting the NGINX/Python webserver on h1...
[sidecar] NOT starting the TCP PEP or sidecar
*** h2 : ('python3 mininet/client.py -n 4000k --http 3 --stdout /dev/null --stderr /home/gina/sidecar/error.log -cc cubic --loss 2 --log-level error -t 10 ',)
head -c 4000k /dev/urandom > /tmp/tmp0o3hf5tf
Data Size: 4000k
HTTP: --http3
RUST_LOG=error sidecurl --http3 --quiche-cc cubic --data-binary @/tmp/tmp0o3hf5tf --insecure https://10.0.1.10:443/ -w "%{time_connect}\t%{time_appconnect}\t%{time_starttransfer}\t\t%{time_total}\t%{exitcode}\t\t%{response_code}\t\t%{size_upload}\t\t%{size_download}\t%{errormsg}\n" --max-time 40 -o /dev/null 2>>/home/gina/sidecar/error.log 
time_connect	time_appconnect	time_starttransfer	time_total	exitcode	response_code	size_upload	size_download	errormsg
0.318905	0.000000	0.318963		11.475854	0		200		4096000		18	
0.164084	0.000000	0.164142		10.525053	0		200		4096000		18	
0.158231	0.000000	0.158282		11.506212	0		200		4096000		18	
0.158247	0.000000	0.158300		10.599266	0		200		4096000		18	
0.158511	0.000000	0.158566		10.733225	0		200		4096000		18	
0.158261	0.000000	0.158317		11.329770	0		200		4096000		18	
0.163986	0.000000	0.164043		14.657959	0		200		4096000		18	
0.164042	0.000000	0.164099		8.688192	0		200		4096000		18	
0.163986	0.000000	0.164043		11.049601	0		200		4096000		18	
0.158258	0.000000	0.158315		6.872938	0		200		4096000		18	
*** Stopping 0 controllers

*** Stopping 2 links
..
*** Stopping 0 switches

*** Stopping 3 hosts
h1 h2 r1 
*** Done
max_queue_size (bytes) = 190000.0
*** Configuring hosts
h1 h2 r1 
[sidecar] Setting congestion control to cubic
[sidecar] tso and gso are OFF
[sidecar] Starting the NGINX/Python webserver on h1...
[sidecar] NOT starting the TCP PEP or sidecar
*** h2 : ('python3 mininet/client.py -n 6000k --http 3 --stdout /dev/null --stderr /home/gina/sidecar/error.log -cc cubic --loss 2 --log-level error -t 10 ',)
head -c 6000k /dev/urandom > /tmp/tmpkioysn3k
Data Size: 6000k
HTTP: --http3
RUST_LOG=error sidecurl --http3 --quiche-cc cubic --data-binary @/tmp/tmpkioysn3k --insecure https://10.0.1.10:443/ -w "%{time_connect}\t%{time_appconnect}\t%{time_starttransfer}\t\t%{time_total}\t%{exitcode}\t\t%{response_code}\t\t%{size_upload}\t\t%{size_download}\t%{errormsg}\n" --max-time 60 -o /dev/null 2>>/home/gina/sidecar/error.log 
time_connect	time_appconnect	time_starttransfer	time_total	exitcode	response_code	size_upload	size_download	errormsg
0.319188	0.000000	0.319245		12.319371	0		200		6144000		18	
0.163890	0.000000	0.163943		17.593734	0		200		6144000		18	
0.163898	0.000000	0.163952		17.111293	0		200		6144000		18	
0.158201	0.000000	0.158253		11.596062	0		200		6144000		18	
0.158247	0.000000	0.158303		11.113455	0		200		6144000		18	
0.163975	0.000000	0.164032		59.999012	28		000		14521		0	Operation timed out after 59999 milliseconds with 0 bytes received
0.158374	0.000000	0.158428		15.990117	0		200		6144000		18	
0.163989	0.000000	0.164041		10.763775	0		200		6144000		18	
0.163835	0.000000	0.163883		14.882830	0		200		6144000		18	
0.164085	0.000000	0.164141		11.802417	0		200		6144000		18	
*** Stopping 0 controllers

*** Stopping 2 links
..
*** Stopping 0 switches

*** Stopping 3 hosts
h1 h2 r1 
*** Done
max_queue_size (bytes) = 190000.0
*** Configuring hosts
h1 h2 r1 
[sidecar] Setting congestion control to cubic
[sidecar] tso and gso are OFF
[sidecar] Starting the NGINX/Python webserver on h1...
[sidecar] NOT starting the TCP PEP or sidecar
*** h2 : ('python3 mininet/client.py -n 8000k --http 3 --stdout /dev/null --stderr /home/gina/sidecar/error.log -cc cubic --loss 2 --log-level error -t 10 ',)
head -c 8000k /dev/urandom > /tmp/tmpa37o2nzi
Data Size: 8000k
HTTP: --http3
RUST_LOG=error sidecurl --http3 --quiche-cc cubic --data-binary @/tmp/tmpa37o2nzi --insecure https://10.0.1.10:443/ -w "%{time_connect}\t%{time_appconnect}\t%{time_starttransfer}\t\t%{time_total}\t%{exitcode}\t\t%{response_code}\t\t%{size_upload}\t\t%{size_download}\t%{errormsg}\n" --max-time 80 -o /dev/null 2>>/home/gina/sidecar/error.log 
time_connect	time_appconnect	time_starttransfer	time_total	exitcode	response_code	size_upload	size_download	errormsg
0.318985	0.000000	0.319043		14.350064	0		200		8192000		18	
0.158272	0.000000	0.158321		79.999012	28		000		14521		0	Operation timed out after 79999 milliseconds with 0 bytes received
0.158343	0.000000	0.158396		14.921774	0		200		8192000		18	
0.163902	0.000000	0.163957		15.094581	0		200		8192000		18	
0.158462	0.000000	0.158519		18.988495	0		200		8192000		18	
0.164018	0.000000	0.164076		15.156164	0		200		8192000		18	
0.158459	0.000000	0.158511		12.843963	0		200		8192000		18	
0.164032	0.000000	0.164088		16.458604	0		200		8192000		18	
0.158230	0.000000	0.158285		14.490256	0		200		8192000		18	
0.158235	0.000000	0.158303		13.297970	0		200		8192000		18	
*** Stopping 0 controllers

*** Stopping 2 links
..
*** Stopping 0 switches

*** Stopping 3 hosts
h1 h2 r1 
*** Done
max_queue_size (bytes) = 190000.0
*** Configuring hosts
h1 h2 r1 
[sidecar] Setting congestion control to cubic
[sidecar] tso and gso are OFF
[sidecar] Starting the NGINX/Python webserver on h1...
[sidecar] NOT starting the TCP PEP or sidecar
*** h2 : ('python3 mininet/client.py -n 10000k --http 3 --stdout /dev/null --stderr /home/gina/sidecar/error.log -cc cubic --loss 2 --log-level error -t 10 ',)
head -c 10000k /dev/urandom > /tmp/tmpaue3tpf_
Data Size: 10000k
HTTP: --http3
RUST_LOG=error sidecurl --http3 --quiche-cc cubic --data-binary @/tmp/tmpaue3tpf_ --insecure https://10.0.1.10:443/ -w "%{time_connect}\t%{time_appconnect}\t%{time_starttransfer}\t\t%{time_total}\t%{exitcode}\t\t%{response_code}\t\t%{size_upload}\t\t%{size_download}\t%{errormsg}\n" --max-time 100 -o /dev/null 2>>/home/gina/sidecar/error.log 
time_connect	time_appconnect	time_starttransfer	time_total	exitcode	response_code	size_upload	size_download	errormsg
1.316101	0.000000	1.316164		18.219548	0		200		10240000		18	
0.164127	0.000000	0.164185		16.873458	0		200		10240000		18	
0.158233	0.000000	0.158289		15.699232	0		200		10240000		18	
0.163985	0.000000	0.164040		16.099540	0		200		10240000		18	
0.164082	0.000000	0.164154		16.249478	0		200		10240000		18	
0.158274	0.000000	0.158330		15.984688	0		200		10240000		18	
0.158206	0.000000	0.158255		14.883049	0		200		10240000		18	
0.158485	0.000000	0.158539		16.771043	0		200		10240000		18	
0.158230	0.000000	0.158286		16.405552	0		200		10240000		18	
0.158242	0.000000	0.158299		16.974994	0		200		10240000		18	
*** Stopping 0 controllers

*** Stopping 2 links
..
*** Stopping 0 switches

*** Stopping 3 hosts
h1 h2 r1 
*** Done
max_queue_size (bytes) = 190000.0
*** Configuring hosts
h1 h2 r1 
[sidecar] Setting congestion control to cubic
[sidecar] tso and gso are OFF
[sidecar] Starting the NGINX/Python webserver on h1...
[sidecar] NOT starting the TCP PEP or sidecar
*** h2 : ('python3 mininet/client.py -n 20000k --http 3 --stdout /dev/null --stderr /home/gina/sidecar/error.log -cc cubic --loss 2 --log-level error -t 10 ',)
head -c 20000k /dev/urandom > /tmp/tmps_xu90h6
Data Size: 20000k
HTTP: --http3
RUST_LOG=error sidecurl --http3 --quiche-cc cubic --data-binary @/tmp/tmps_xu90h6 --insecure https://10.0.1.10:443/ -w "%{time_connect}\t%{time_appconnect}\t%{time_starttransfer}\t\t%{time_total}\t%{exitcode}\t\t%{response_code}\t\t%{size_upload}\t\t%{size_download}\t%{errormsg}\n" --max-time 200 -o /dev/null 2>>/home/gina/sidecar/error.log 
time_connect	time_appconnect	time_starttransfer	time_total	exitcode	response_code	size_upload	size_download	errormsg
0.324529	0.000000	0.324588		26.599590	0		200		20480000		18	
0.158247	0.000000	0.158306		25.894162	0		200		20480000		18	
0.158218	0.000000	0.158269		31.444449	0		200		20480000		18	
0.158244	0.000000	0.158301		25.055165	0		200		20480000		18	
0.158443	0.000000	0.158496		25.275474	0		200		20480000		18	
0.158496	0.000000	0.158549		24.783743	0		200		20480000		18	
0.158263	0.000000	0.158319		24.835616	0		200		20480000		18	
0.158281	0.000000	0.158336		25.442623	0		200		20480000		18	
0.163957	0.000000	0.164015		25.356726	0		200		20480000		18	
1.157357	0.000000	1.157415		27.600904	0		200		20480000		18	
*** Stopping 0 controllers

*** Stopping 2 links
..
*** Stopping 0 switches

*** Stopping 3 hosts
h1 h2 r1 
*** Done
max_queue_size (bytes) = 190000.0
*** Configuring hosts
h1 h2 r1 
[sidecar] Setting congestion control to cubic
[sidecar] tso and gso are OFF
[sidecar] Starting the NGINX/Python webserver on h1...
[sidecar] NOT starting the TCP PEP or sidecar
*** h2 : ('python3 mininet/client.py -n 30000k --http 3 --stdout /dev/null --stderr /home/gina/sidecar/error.log -cc cubic --loss 2 --log-level error -t 10 ',)
head -c 30000k /dev/urandom > /tmp/tmpwpa4lalb
Data Size: 30000k
HTTP: --http3
RUST_LOG=error sidecurl --http3 --quiche-cc cubic --data-binary @/tmp/tmpwpa4lalb --insecure https://10.0.1.10:443/ -w "%{time_connect}\t%{time_appconnect}\t%{time_starttransfer}\t\t%{time_total}\t%{exitcode}\t\t%{response_code}\t\t%{size_upload}\t\t%{size_download}\t%{errormsg}\n" --max-time 300 -o /dev/null 2>>/home/gina/sidecar/error.log 
time_connect	time_appconnect	time_starttransfer	time_total	exitcode	response_code	size_upload	size_download	errormsg
1.326139	0.000000	1.326197		37.286612	0		200		30720000		18	
0.158438	0.000000	0.158498		38.218886	0		200		30720000		18	
1.464634	0.000000	1.464693		37.425470	0		200		30720000		18	
0.158288	0.000000	0.158348		35.331589	0		200		30720000		18	
0.158270	0.000000	0.158328		33.521835	0		200		30720000		18	
0.158260	0.000000	0.158316		35.404130	0		200		30720000		18	
0.164011	0.000000	0.164068		39.249621	0		200		30720000		18	
0.164107	0.000000	0.164167		35.741048	0		200		30720000		18	
0.158277	0.000000	0.158333		35.765420	0		200		30720000		18	
0.158254	0.000000	0.158311		35.314306	0		200		30720000		18	
*** Stopping 0 controllers

*** Stopping 2 links
..
*** Stopping 0 switches

*** Stopping 3 hosts
h1 h2 r1 
*** Done
max_queue_size (bytes) = 190000.0
*** Configuring hosts
h1 h2 r1 
[sidecar] Setting congestion control to cubic
[sidecar] tso and gso are OFF
[sidecar] Starting the NGINX/Python webserver on h1...
[sidecar] NOT starting the TCP PEP or sidecar
*** h2 : ('python3 mininet/client.py -n 40000k --http 3 --stdout /dev/null --stderr /home/gina/sidecar/error.log -cc cubic --loss 2 --log-level error -t 10 ',)
head -c 40000k /dev/urandom > /tmp/tmpf1_ua2dk
Data Size: 40000k
HTTP: --http3
RUST_LOG=error sidecurl --http3 --quiche-cc cubic --data-binary @/tmp/tmpf1_ua2dk --insecure https://10.0.1.10:443/ -w "%{time_connect}\t%{time_appconnect}\t%{time_starttransfer}\t\t%{time_total}\t%{exitcode}\t\t%{response_code}\t\t%{size_upload}\t\t%{size_download}\t%{errormsg}\n" --max-time 400 -o /dev/null 2>>/home/gina/sidecar/error.log 
time_connect	time_appconnect	time_starttransfer	time_total	exitcode	response_code	size_upload	size_download	errormsg
0.318924	0.000000	0.318980		48.180269	0		200		40960000		18	
0.164078	0.000000	0.164137		43.907940	0		200		40960000		18	
0.158436	0.000000	0.158490		52.480315	0		200		40960000		18	
0.158513	0.000000	0.158571		43.785106	0		200		40960000		18	
0.158523	0.000000	0.158581		44.696296	0		200		40960000		18	
0.164174	0.000000	0.164234		45.164853	0		200		40960000		18	
0.158297	0.000000	0.158354		43.807018	0		200		40960000		18	
0.164083	0.000000	0.164139		44.158320	0		200		40960000		18	
0.164072	0.000000	0.164129		45.031756	0		200		40960000		18	
0.158446	0.000000	0.158501		56.524431	0		200		40960000		18	
*** Stopping 0 controllers

*** Stopping 2 links
..
*** Stopping 0 switches

*** Stopping 3 hosts
h1 h2 r1 
*** Done
max_queue_size (bytes) = 190000.0
*** Configuring hosts
h1 h2 r1 
[sidecar] Setting congestion control to cubic
[sidecar] tso and gso are OFF
[sidecar] Starting the NGINX/Python webserver on h1...
[sidecar] NOT starting the TCP PEP or sidecar
*** h2 : ('python3 mininet/client.py -n 50000k --http 3 --stdout /dev/null --stderr /home/gina/sidecar/error.log -cc cubic --loss 2 --log-level error -t 10 ',)
head -c 50000k /dev/urandom > /tmp/tmph9t_xyrw
Data Size: 50000k
HTTP: --http3
RUST_LOG=error sidecurl --http3 --quiche-cc cubic --data-binary @/tmp/tmph9t_xyrw --insecure https://10.0.1.10:443/ -w "%{time_connect}\t%{time_appconnect}\t%{time_starttransfer}\t\t%{time_total}\t%{exitcode}\t\t%{response_code}\t\t%{size_upload}\t\t%{size_download}\t%{errormsg}\n" --max-time 500 -o /dev/null 2>>/home/gina/sidecar/error.log 
time_connect	time_appconnect	time_starttransfer	time_total	exitcode	response_code	size_upload	size_download	errormsg
0.324291	0.000000	0.324347		56.960169	0		200		51200000		18	
0.158184	0.000000	0.158236		53.648956	0		200		51200000		18	
0.158502	0.000000	0.158560		55.153156	0		200		51200000		18	
0.164072	0.000000	0.164128		53.101767	0		200		51200000		18	
0.158341	0.000000	0.158402		54.847816	0		200		51200000		18	
0.164075	0.000000	0.164133		55.555135	0		200		51200000		18	
0.164105	0.000000	0.164163		59.257324	0		200		51200000		18	
0.158534	0.000000	0.158590		55.041143	0		200		51200000		18	
0.158274	0.000000	0.158333		54.433121	0		200		51200000		18	
0.158227	0.000000	0.158282		58.857636	0		200		51200000		18	
*** Stopping 0 controllers

*** Stopping 2 links
..
*** Stopping 0 switches

*** Stopping 3 hosts
h1 h2 r1 
*** Done
max_queue_size (bytes) = 190000.0
*** Configuring hosts
h1 h2 r1 
[sidecar] Setting congestion control to cubic
[sidecar] tso and gso are OFF
[sidecar] Starting the NGINX/Python webserver on h1...
[sidecar] NOT starting the TCP PEP or sidecar
*** h2 : ('python3 mininet/client.py -n 200k --http 3 --stdout /dev/null --stderr /home/gina/sidecar/error.log -cc cubic --loss 2 --log-level error -t 2 ',)
head -c 200k /dev/urandom > /tmp/tmpubbzxf_u
Data Size: 200k
HTTP: --http3
RUST_LOG=error sidecurl --http3 --quiche-cc cubic --data-binary @/tmp/tmpubbzxf_u --insecure https://10.0.1.10:443/ -w "%{time_connect}\t%{time_appconnect}\t%{time_starttransfer}\t\t%{time_total}\t%{exitcode}\t\t%{response_code}\t\t%{size_upload}\t\t%{size_download}\t%{errormsg}\n" --max-time 15 -o /dev/null 2>>/home/gina/sidecar/error.log 
time_connect	time_appconnect	time_starttransfer	time_total	exitcode	response_code	size_upload	size_download	errormsg
0.319021	0.000000	0.319078		1.261165	0		200		204800		18	
1.468645	0.000000	1.468705		3.145045	0		200		204800		18	
*** Stopping 0 controllers

*** Stopping 2 links
..
*** Stopping 0 switches

*** Stopping 3 hosts
h1 h2 r1 
*** Done
max_queue_size (bytes) = 190000.0
*** Configuring hosts
h1 h2 r1 
[sidecar] Setting congestion control to cubic
[sidecar] tso and gso are OFF
[sidecar] Starting the NGINX/Python webserver on h1...
[sidecar] NOT starting the TCP PEP or sidecar
*** h2 : ('python3 mininet/client.py -n 600k --http 3 --stdout /dev/null --stderr /home/gina/sidecar/error.log -cc cubic --loss 2 --log-level error -t 1 ',)
head -c 600k /dev/urandom > /tmp/tmp7wspn79z
Data Size: 600k
HTTP: --http3
RUST_LOG=error sidecurl --http3 --quiche-cc cubic --data-binary @/tmp/tmp7wspn79z --insecure https://10.0.1.10:443/ -w "%{time_connect}\t%{time_appconnect}\t%{time_starttransfer}\t\t%{time_total}\t%{exitcode}\t\t%{response_code}\t\t%{size_upload}\t\t%{size_download}\t%{errormsg}\n" --max-time 15 -o /dev/null 2>>/home/gina/sidecar/error.log 
time_connect	time_appconnect	time_starttransfer	time_total	exitcode	response_code	size_upload	size_download	errormsg
0.319021	0.000000	0.319076		2.502365	0		200		614400		18	
*** Stopping 0 controllers

*** Stopping 2 links
..
*** Stopping 0 switches

*** Stopping 3 hosts
h1 h2 r1 
*** Done
max_queue_size (bytes) = 190000.0
*** Configuring hosts
h1 h2 r1 
[sidecar] Setting congestion control to cubic
[sidecar] tso and gso are OFF
[sidecar] Starting the NGINX/Python webserver on h1...
[sidecar] NOT starting the TCP PEP or sidecar
*** h2 : ('python3 mininet/client.py -n 1000k --http 3 --stdout /dev/null --stderr /home/gina/sidecar/error.log -cc cubic --loss 2 --log-level error -t 1 ',)
head -c 1000k /dev/urandom > /tmp/tmpkvlphfe6
Data Size: 1000k
HTTP: --http3
RUST_LOG=error sidecurl --http3 --quiche-cc cubic --data-binary @/tmp/tmpkvlphfe6 --insecure https://10.0.1.10:443/ -w "%{time_connect}\t%{time_appconnect}\t%{time_starttransfer}\t\t%{time_total}\t%{exitcode}\t\t%{response_code}\t\t%{size_upload}\t\t%{size_download}\t%{errormsg}\n" --max-time 15 -o /dev/null 2>>/home/gina/sidecar/error.log 
time_connect	time_appconnect	time_starttransfer	time_total	exitcode	response_code	size_upload	size_download	errormsg
1.316120	0.000000	1.316176		8.659109	0		200		1024000		18	
*** Stopping 0 controllers

*** Stopping 2 links
..
*** Stopping 0 switches

*** Stopping 3 hosts
h1 h2 r1 
*** Done
max_queue_size (bytes) = 190000.0
*** Configuring hosts
h1 h2 r1 
[sidecar] Setting congestion control to cubic
[sidecar] tso and gso are OFF
[sidecar] Starting the NGINX/Python webserver on h1...
[sidecar] NOT starting the TCP PEP or sidecar
*** h2 : ('python3 mininet/client.py -n 6000k --http 3 --stdout /dev/null --stderr /home/gina/sidecar/error.log -cc cubic --loss 2 --log-level error -t 1 ',)
head -c 6000k /dev/urandom > /tmp/tmpm97bfunb
Data Size: 6000k
HTTP: --http3
RUST_LOG=error sidecurl --http3 --quiche-cc cubic --data-binary @/tmp/tmpm97bfunb --insecure https://10.0.1.10:443/ -w "%{time_connect}\t%{time_appconnect}\t%{time_starttransfer}\t\t%{time_total}\t%{exitcode}\t\t%{response_code}\t\t%{size_upload}\t\t%{size_download}\t%{errormsg}\n" --max-time 60 -o /dev/null 2>>/home/gina/sidecar/error.log 
time_connect	time_appconnect	time_starttransfer	time_total	exitcode	response_code	size_upload	size_download	errormsg
0.319131	0.000000	0.319190		11.086894	0		200		6144000		18	
*** Stopping 0 controllers

*** Stopping 2 links
..
*** Stopping 0 switches

*** Stopping 3 hosts
h1 h2 r1 
*** Done
max_queue_size (bytes) = 190000.0
*** Configuring hosts
h1 h2 r1 
[sidecar] Setting congestion control to cubic
[sidecar] tso and gso are OFF
[sidecar] Starting the NGINX/Python webserver on h1...
[sidecar] NOT starting the TCP PEP or sidecar
*** h2 : ('python3 mininet/client.py -n 8000k --http 3 --stdout /dev/null --stderr /home/gina/sidecar/error.log -cc cubic --loss 2 --log-level error -t 1 ',)
head -c 8000k /dev/urandom > /tmp/tmprq5rxvqk
Data Size: 8000k
HTTP: --http3
RUST_LOG=error sidecurl --http3 --quiche-cc cubic --data-binary @/tmp/tmprq5rxvqk --insecure https://10.0.1.10:443/ -w "%{time_connect}\t%{time_appconnect}\t%{time_starttransfer}\t\t%{time_total}\t%{exitcode}\t\t%{response_code}\t\t%{size_upload}\t\t%{size_download}\t%{errormsg}\n" --max-time 80 -o /dev/null 2>>/home/gina/sidecar/error.log 
time_connect	time_appconnect	time_starttransfer	time_total	exitcode	response_code	size_upload	size_download	errormsg
0.318920	0.000000	0.318976		18.253436	0		200		8192000		18	
*** Stopping 0 controllers

*** Stopping 2 links
..
*** Stopping 0 switches

*** Stopping 3 hosts
h1 h2 r1 
*** Done
max_queue_size (bytes) = 190000.0
*** Configuring hosts
h1 h2 r1 
[sidecar] Setting congestion control to cubic
[sidecar] tso and gso are OFF
[sidecar] Starting the NGINX/Python webserver on h1...
[sidecar] NOT starting the TCP PEP or sidecar
*** h2 : ('python3 mininet/client.py -n 3000k --http 3 --stdout /dev/null --stderr /home/gina/sidecar/error.log -cc cubic --loss 2 --log-level error -t 10 ',)
head -c 3000k /dev/urandom > /tmp/tmpfppoxybz
Data Size: 3000k
HTTP: --http3
RUST_LOG=error sidecurl --http3 --quiche-cc cubic --data-binary @/tmp/tmpfppoxybz --insecure https://10.0.1.10:443/ -w "%{time_connect}\t%{time_appconnect}\t%{time_starttransfer}\t\t%{time_total}\t%{exitcode}\t\t%{response_code}\t\t%{size_upload}\t\t%{size_download}\t%{errormsg}\n" --max-time 30 -o /dev/null 2>>/home/gina/sidecar/error.log 
time_connect	time_appconnect	time_starttransfer	time_total	exitcode	response_code	size_upload	size_download	errormsg
0.330077	0.000000	0.330136		11.855380	0		200		3072000		18	
0.158449	0.000000	0.158505		8.787754	0		200		3072000		18	
0.158495	0.000000	0.158551		7.399798	0		200		3072000		18	
0.158232	0.000000	0.158285		9.157979	0		200		3072000		18	
0.158555	0.000000	0.158610		10.019144	0		200		3072000		18	
0.164007	0.000000	0.164062		7.998716	0		200		3072000		18	
0.158269	0.000000	0.158326		6.438337	0		200		3072000		18	
0.158421	0.000000	0.158480		6.729727	0		200		3072000		18	
0.158187	0.000000	0.158240		8.186690	0		200		3072000		18	
0.158425	0.000000	0.158480		12.994097	0		200		3072000		18	
*** Stopping 0 controllers

*** Stopping 2 links
..
*** Stopping 0 switches

*** Stopping 3 hosts
h1 h2 r1 
*** Done
max_queue_size (bytes) = 190000.0
*** Configuring hosts
h1 h2 r1 
[sidecar] Setting congestion control to cubic
[sidecar] tso and gso are OFF
[sidecar] Starting the NGINX/Python webserver on h1...
[sidecar] NOT starting the TCP PEP or sidecar
*** h2 : ('python3 mininet/client.py -n 5000k --http 3 --stdout /dev/null --stderr /home/gina/sidecar/error.log -cc cubic --loss 2 --log-level error -t 10 ',)
head -c 5000k /dev/urandom > /tmp/tmp1sm9zoo_
Data Size: 5000k
HTTP: --http3
RUST_LOG=error sidecurl --http3 --quiche-cc cubic --data-binary @/tmp/tmp1sm9zoo_ --insecure https://10.0.1.10:443/ -w "%{time_connect}\t%{time_appconnect}\t%{time_starttransfer}\t\t%{time_total}\t%{exitcode}\t\t%{response_code}\t\t%{size_upload}\t\t%{size_download}\t%{errormsg}\n" --max-time 50 -o /dev/null 2>>/home/gina/sidecar/error.log 
time_connect	time_appconnect	time_starttransfer	time_total	exitcode	response_code	size_upload	size_download	errormsg
0.319500	0.000000	0.319556		12.267312	0		200		5120000		18	
0.158259	0.000000	0.158316		13.222634	0		200		5120000		18	
0.542121	0.000000	0.542178		14.799082	0		200		5120000		18	
0.158466	0.000000	0.158521		11.808189	0		200		5120000		18	
0.158510	0.000000	0.158570		49.999011	28		000		14521		0	Operation timed out after 49999 milliseconds with 0 bytes received
0.158385	0.000000	0.158442		12.608736	0		200		5120000		18	
0.158266	0.000000	0.158321		49.999013	28		000		5120000		0	Operation timed out after 49999 milliseconds with 0 bytes received
0.158340	0.000000	0.158395		15.408347	0		200		5120000		18	
0.158480	0.000000	0.158537		10.369337	0		200		5120000		18	
0.158277	0.000000	0.158334		11.907994	0		200		5120000		18	
*** Stopping 0 controllers

*** Stopping 2 links
..
*** Stopping 0 switches

*** Stopping 3 hosts
h1 h2 r1 
*** Done
max_queue_size (bytes) = 190000.0
*** Configuring hosts
h1 h2 r1 
[sidecar] Setting congestion control to cubic
[sidecar] tso and gso are OFF
[sidecar] Starting the NGINX/Python webserver on h1...
[sidecar] NOT starting the TCP PEP or sidecar
*** h2 : ('python3 mininet/client.py -n 7000k --http 3 --stdout /dev/null --stderr /home/gina/sidecar/error.log -cc cubic --loss 2 --log-level error -t 10 ',)
head -c 7000k /dev/urandom > /tmp/tmpwavq_fx2
Data Size: 7000k
HTTP: --http3
RUST_LOG=error sidecurl --http3 --quiche-cc cubic --data-binary @/tmp/tmpwavq_fx2 --insecure https://10.0.1.10:443/ -w "%{time_connect}\t%{time_appconnect}\t%{time_starttransfer}\t\t%{time_total}\t%{exitcode}\t\t%{response_code}\t\t%{size_upload}\t\t%{size_download}\t%{errormsg}\n" --max-time 70 -o /dev/null 2>>/home/gina/sidecar/error.log 
time_connect	time_appconnect	time_starttransfer	time_total	exitcode	response_code	size_upload	size_download	errormsg
0.324309	0.000000	0.324366		13.473728	0		200		7168000		18	
0.158285	0.000000	0.158342		13.509601	0		200		7168000		18	
0.158284	0.000000	0.158343		13.063062	0		200		7168000		18	
0.158546	0.000000	0.158602		14.141202	0		200		7168000		18	
0.158271	0.000000	0.158327		11.906162	0		200		7168000		18	
0.164073	0.000000	0.164132		16.109389	0		200		7168000		18	
0.158273	0.000000	0.158328		12.185238	0		200		7168000		18	
0.158186	0.000000	0.158240		12.252688	0		200		7168000		18	
0.158251	0.000000	0.158307		13.922266	0		200		7168000		18	
0.158235	0.000000	0.158291		14.013968	0		200		7168000		18	
*** Stopping 0 controllers

*** Stopping 2 links
..
*** Stopping 0 switches

*** Stopping 3 hosts
h1 h2 r1 
*** Done
max_queue_size (bytes) = 190000.0
*** Configuring hosts
h1 h2 r1 
[sidecar] Setting congestion control to cubic
[sidecar] tso and gso are OFF
[sidecar] Starting the NGINX/Python webserver on h1...
[sidecar] NOT starting the TCP PEP or sidecar
*** h2 : ('python3 mininet/client.py -n 9000k --http 3 --stdout /dev/null --stderr /home/gina/sidecar/error.log -cc cubic --loss 2 --log-level error -t 10 ',)
head -c 9000k /dev/urandom > /tmp/tmp8mznfg95
Data Size: 9000k
HTTP: --http3
RUST_LOG=error sidecurl --http3 --quiche-cc cubic --data-binary @/tmp/tmp8mznfg95 --insecure https://10.0.1.10:443/ -w "%{time_connect}\t%{time_appconnect}\t%{time_starttransfer}\t\t%{time_total}\t%{exitcode}\t\t%{response_code}\t\t%{size_upload}\t\t%{size_download}\t%{errormsg}\n" --max-time 90 -o /dev/null 2>>/home/gina/sidecar/error.log 
time_connect	time_appconnect	time_starttransfer	time_total	exitcode	response_code	size_upload	size_download	errormsg
0.318933	0.000000	0.318990		15.895604	0		200		9216000		18	
0.158484	0.000000	0.158541		19.564142	0		200		9216000		18	
0.158543	0.000000	0.158600		13.899317	0		200		9216000		18	
2.308417	0.000000	2.308475		17.532318	0		200		9216000		18	
0.158271	0.000000	0.158328		14.268238	0		200		9216000		18	
0.158286	0.000000	0.158344		12.780191	0		200		9216000		18	
0.163949	0.000000	0.164007		14.840043	0		200		9216000		18	
0.158504	0.000000	0.158559		19.094878	0		200		9216000		18	
0.158216	0.000000	0.158271		18.645833	0		200		9216000		18	
0.158229	0.000000	0.158287		16.100828	0		200		9216000		18	
*** Stopping 0 controllers

*** Stopping 2 links
..
*** Stopping 0 switches

*** Stopping 3 hosts
h1 h2 r1 
*** Done
max_queue_size (bytes) = 190000.0
*** Configuring hosts
h1 h2 r1 
[sidecar] Setting congestion control to cubic
[sidecar] tso and gso are OFF
[sidecar] Starting the NGINX/Python webserver on h1...
[sidecar] NOT starting the TCP PEP or sidecar
*** h2 : ('python3 mininet/client.py -n 12000k --http 3 --stdout /dev/null --stderr /home/gina/sidecar/error.log -cc cubic --loss 2 --log-level error -t 10 ',)
head -c 12000k /dev/urandom > /tmp/tmpiwqgcscz
Data Size: 12000k
HTTP: --http3
RUST_LOG=error sidecurl --http3 --quiche-cc cubic --data-binary @/tmp/tmpiwqgcscz --insecure https://10.0.1.10:443/ -w "%{time_connect}\t%{time_appconnect}\t%{time_starttransfer}\t\t%{time_total}\t%{exitcode}\t\t%{response_code}\t\t%{size_upload}\t\t%{size_download}\t%{errormsg}\n" --max-time 120 -o /dev/null 2>>/home/gina/sidecar/error.log 
time_connect	time_appconnect	time_starttransfer	time_total	exitcode	response_code	size_upload	size_download	errormsg
0.330245	0.000000	0.330301		18.608540	0		200		12288000		18	
0.158264	0.000000	0.158320		18.543937	0		200		12288000		18	
0.163989	0.000000	0.164049		18.715492	0		200		12288000		18	
0.163994	0.000000	0.164051		18.088999	0		200		12288000		18	
0.164071	0.000000	0.164128		18.547156	0		200		12288000		18	
0.164092	0.000000	0.164151		18.510309	0		200		12288000		18	
0.164002	0.000000	0.164058		119.999012	28		000		14521		0	Operation timed out after 119999 milliseconds with 0 bytes received
0.164157	0.000000	0.164214		17.710371	0		200		12288000		18	
0.164020	0.000000	0.164079		18.571332	0		200		12288000		18	
0.542999	0.000000	0.543054		18.927140	0		200		12288000		18	
*** Stopping 0 controllers

*** Stopping 2 links
..
*** Stopping 0 switches

*** Stopping 3 hosts
h1 h2 r1 
*** Done
max_queue_size (bytes) = 190000.0
*** Configuring hosts
h1 h2 r1 
[sidecar] Setting congestion control to cubic
[sidecar] tso and gso are OFF
[sidecar] Starting the NGINX/Python webserver on h1...
[sidecar] NOT starting the TCP PEP or sidecar
*** h2 : ('python3 mininet/client.py -n 14000k --http 3 --stdout /dev/null --stderr /home/gina/sidecar/error.log -cc cubic --loss 2 --log-level error -t 10 ',)
head -c 14000k /dev/urandom > /tmp/tmptijpkor1
Data Size: 14000k
HTTP: --http3
RUST_LOG=error sidecurl --http3 --quiche-cc cubic --data-binary @/tmp/tmptijpkor1 --insecure https://10.0.1.10:443/ -w "%{time_connect}\t%{time_appconnect}\t%{time_starttransfer}\t\t%{time_total}\t%{exitcode}\t\t%{response_code}\t\t%{size_upload}\t\t%{size_download}\t%{errormsg}\n" --max-time 140 -o /dev/null 2>>/home/gina/sidecar/error.log 
time_connect	time_appconnect	time_starttransfer	time_total	exitcode	response_code	size_upload	size_download	errormsg
0.319072	0.000000	0.319131		21.900962	0		200		14336000		18	
0.158501	0.000000	0.158558		19.566703	0		200		14336000		18	
0.163999	0.000000	0.164054		21.420450	0		200		14336000		18	
0.158288	0.000000	0.158345		20.514207	0		200		14336000		18	
0.158239	0.000000	0.158297		18.383331	0		200		14336000		18	
0.158293	0.000000	0.158351		21.340269	0		200		14336000		18	
0.158279	0.000000	0.158336		139.999009	28		000		14336000		0	Operation timed out after 139999 milliseconds with 0 bytes received
0.158616	0.000000	0.158671		18.414106	0		200		14336000		18	
0.158255	0.000000	0.158313		22.564053	0		200		14336000		18	
0.158265	0.000000	0.158323		19.844586	0		200		14336000		18	
*** Stopping 0 controllers

*** Stopping 2 links
..
*** Stopping 0 switches

*** Stopping 3 hosts
h1 h2 r1 
*** Done
max_queue_size (bytes) = 190000.0
*** Configuring hosts
h1 h2 r1 
[sidecar] Setting congestion control to cubic
[sidecar] tso and gso are OFF
[sidecar] Starting the NGINX/Python webserver on h1...
[sidecar] NOT starting the TCP PEP or sidecar
*** h2 : ('python3 mininet/client.py -n 16000k --http 3 --stdout /dev/null --stderr /home/gina/sidecar/error.log -cc cubic --loss 2 --log-level error -t 10 ',)
head -c 16000k /dev/urandom > /tmp/tmpkdapfo8s
Data Size: 16000k
HTTP: --http3
RUST_LOG=error sidecurl --http3 --quiche-cc cubic --data-binary @/tmp/tmpkdapfo8s --insecure https://10.0.1.10:443/ -w "%{time_connect}\t%{time_appconnect}\t%{time_starttransfer}\t\t%{time_total}\t%{exitcode}\t\t%{response_code}\t\t%{size_upload}\t\t%{size_download}\t%{errormsg}\n" --max-time 160 -o /dev/null 2>>/home/gina/sidecar/error.log 
time_connect	time_appconnect	time_starttransfer	time_total	exitcode	response_code	size_upload	size_download	errormsg
0.318935	0.000000	0.318992		21.328855	0		200		16384000		18	
0.158525	0.000000	0.158583		27.214619	0		200		16384000		18	
0.158226	0.000000	0.158281		24.715847	0		200		16384000		18	
0.164066	0.000000	0.164124		21.545882	0		200		16384000		18	
0.158265	0.000000	0.158321		22.377145	0		200		16384000		18	
0.158285	0.000000	0.158344		16.356411	0		200		16384000		18	
0.158256	0.000000	0.158314		22.554338	0		200		16384000		18	
0.164077	0.000000	0.164130		22.595266	0		200		16384000		18	
0.158296	0.000000	0.158353		22.265160	0		200		16384000		18	
0.158262	0.000000	0.158320		22.512871	0		200		16384000		18	
*** Stopping 0 controllers

*** Stopping 2 links
..
*** Stopping 0 switches

*** Stopping 3 hosts
h1 h2 r1 
*** Done
max_queue_size (bytes) = 190000.0
*** Configuring hosts
h1 h2 r1 
[sidecar] Setting congestion control to cubic
[sidecar] tso and gso are OFF
[sidecar] Starting the NGINX/Python webserver on h1...
[sidecar] NOT starting the TCP PEP or sidecar
*** h2 : ('python3 mininet/client.py -n 18000k --http 3 --stdout /dev/null --stderr /home/gina/sidecar/error.log -cc cubic --loss 2 --log-level error -t 10 ',)
head -c 18000k /dev/urandom > /tmp/tmpbl0wd0ur
Data Size: 18000k
HTTP: --http3
RUST_LOG=error sidecurl --http3 --quiche-cc cubic --data-binary @/tmp/tmpbl0wd0ur --insecure https://10.0.1.10:443/ -w "%{time_connect}\t%{time_appconnect}\t%{time_starttransfer}\t\t%{time_total}\t%{exitcode}\t\t%{response_code}\t\t%{size_upload}\t\t%{size_download}\t%{errormsg}\n" --max-time 180 -o /dev/null 2>>/home/gina/sidecar/error.log 
time_connect	time_appconnect	time_starttransfer	time_total	exitcode	response_code	size_upload	size_download	errormsg
0.324330	0.000000	0.324385		25.438446	0		200		18432000		18	
0.164011	0.000000	0.164067		24.690774	0		200		18432000		18	
0.158541	0.000000	0.158596		24.596079	0		200		18432000		18	
0.158238	0.000000	0.158296		23.378597	0		200		18432000		18	
1.157317	0.000000	1.157374		25.448316	0		200		18432000		18	
0.158288	0.000000	0.158344		27.263084	0		200		18432000		18	
1.157367	0.000000	1.157424		25.459250	0		200		18432000		18	
0.158265	0.000000	0.158318		22.440119	0		200		18432000		18	
0.164043	0.000000	0.164099		23.871428	0		200		18432000		18	
0.158276	0.000000	0.158334		28.209163	0		200		18432000		18	
*** Stopping 0 controllers

*** Stopping 2 links
..
*** Stopping 0 switches

*** Stopping 3 hosts
h1 h2 r1 
*** Done
max_queue_size (bytes) = 190000.0
*** Configuring hosts
h1 h2 r1 
[sidecar] Setting congestion control to cubic
[sidecar] tso and gso are OFF
[sidecar] Starting the NGINX/Python webserver on h1...
[sidecar] NOT starting the TCP PEP or sidecar
*** h2 : ('python3 mininet/client.py -n 25000k --http 3 --stdout /dev/null --stderr /home/gina/sidecar/error.log -cc cubic --loss 2 --log-level error -t 10 ',)
head -c 25000k /dev/urandom > /tmp/tmpism_c7fc
Data Size: 25000k
HTTP: --http3
RUST_LOG=error sidecurl --http3 --quiche-cc cubic --data-binary @/tmp/tmpism_c7fc --insecure https://10.0.1.10:443/ -w "%{time_connect}\t%{time_appconnect}\t%{time_starttransfer}\t\t%{time_total}\t%{exitcode}\t\t%{response_code}\t\t%{size_upload}\t\t%{size_download}\t%{errormsg}\n" --max-time 250 -o /dev/null 2>>/home/gina/sidecar/error.log 
time_connect	time_appconnect	time_starttransfer	time_total	exitcode	response_code	size_upload	size_download	errormsg
0.318954	0.000000	0.319016		29.972579	0		200		25600000		18	
0.158481	0.000000	0.158537		30.059697	0		200		25600000		18	
0.164058	0.000000	0.164118		30.867760	0		200		25600000		18	
0.158261	0.000000	0.158318		31.073283	0		200		25600000		18	
0.158302	0.000000	0.158361		31.127032	0		200		25600000		18	
0.158354	0.000000	0.158410		29.782027	0		200		25600000		18	
0.163969	0.000000	0.164027		30.767599	0		200		25600000		18	
0.158503	0.000000	0.158560		30.996683	0		200		25600000		18	
0.163948	0.000000	0.164003		249.999014	28		000		25600000		0	Operation timed out after 249999 milliseconds with 0 bytes received
0.158384	0.000000	0.158438		29.035153	0		200		25600000		18	
*** Stopping 0 controllers

*** Stopping 2 links
..
*** Stopping 0 switches

*** Stopping 3 hosts
h1 h2 r1 
*** Done
max_queue_size (bytes) = 190000.0
*** Configuring hosts
h1 h2 r1 
[sidecar] Setting congestion control to cubic
[sidecar] tso and gso are OFF
[sidecar] Starting the NGINX/Python webserver on h1...
[sidecar] NOT starting the TCP PEP or sidecar
*** h2 : ('python3 mininet/client.py -n 35000k --http 3 --stdout /dev/null --stderr /home/gina/sidecar/error.log -cc cubic --loss 2 --log-level error -t 10 ',)
head -c 35000k /dev/urandom > /tmp/tmpv7zvs5qq
Data Size: 35000k
HTTP: --http3
RUST_LOG=error sidecurl --http3 --quiche-cc cubic --data-binary @/tmp/tmpv7zvs5qq --insecure https://10.0.1.10:443/ -w "%{time_connect}\t%{time_appconnect}\t%{time_starttransfer}\t\t%{time_total}\t%{exitcode}\t\t%{response_code}\t\t%{size_upload}\t\t%{size_download}\t%{errormsg}\n" --max-time 350 -o /dev/null 2>>/home/gina/sidecar/error.log 
time_connect	time_appconnect	time_starttransfer	time_total	exitcode	response_code	size_upload	size_download	errormsg
0.318911	0.000000	0.318967		43.587285	0		200		35840000		18	
1.163067	0.000000	1.163124		41.887874	0		200		35840000		18	
0.158251	0.000000	0.158306		39.307342	0		200		35840000		18	
0.158268	0.000000	0.158332		349.999014	28		000		35840000		0	Operation timed out after 349999 milliseconds with 0 bytes received
0.164165	0.000000	0.164224		41.394286	0		200		35840000		18	
0.158298	0.000000	0.158356		39.077561	0		200		35840000		18	
0.158290	0.000000	0.158348		39.090621	0		200		35840000		18	
0.158268	0.000000	0.158325		42.242794	0		200		35840000		18	
0.158495	0.000000	0.158552		38.180640	0		200		35840000		18	
0.158307	0.000000	0.158364		40.492643	0		200		35840000		18	
*** Stopping 0 controllers

*** Stopping 2 links
..
*** Stopping 0 switches

*** Stopping 3 hosts
h1 h2 r1 
*** Done
max_queue_size (bytes) = 190000.0
*** Configuring hosts
h1 h2 r1 
[sidecar] Setting congestion control to cubic
[sidecar] tso and gso are OFF
[sidecar] Starting the NGINX/Python webserver on h1...
[sidecar] NOT starting the TCP PEP or sidecar
*** h2 : ('python3 mininet/client.py -n 45000k --http 3 --stdout /dev/null --stderr /home/gina/sidecar/error.log -cc cubic --loss 2 --log-level error -t 10 ',)
head -c 45000k /dev/urandom > /tmp/tmpsaavnn2j
Data Size: 45000k
HTTP: --http3
RUST_LOG=error sidecurl --http3 --quiche-cc cubic --data-binary @/tmp/tmpsaavnn2j --insecure https://10.0.1.10:443/ -w "%{time_connect}\t%{time_appconnect}\t%{time_starttransfer}\t\t%{time_total}\t%{exitcode}\t\t%{response_code}\t\t%{size_upload}\t\t%{size_download}\t%{errormsg}\n" --max-time 450 -o /dev/null 2>>/home/gina/sidecar/error.log 
time_connect	time_appconnect	time_starttransfer	time_total	exitcode	response_code	size_upload	size_download	errormsg
0.324232	0.000000	0.324290		50.582407	0		200		46080000		18	
0.158551	0.000000	0.158610		49.101910	0		200		46080000		18	
0.158278	0.000000	0.158334		48.557553	0		200		46080000		18	
0.164083	0.000000	0.164141		54.811220	0		200		46080000		18	
0.163960	0.000000	0.164018		50.136666	0		200		46080000		18	
0.164059	0.000000	0.164118		49.344635	0		200		46080000		18	
0.164030	0.000000	0.164086		49.468555	0		200		46080000		18	
1.157350	0.000000	1.157406		51.343888	0		200		46080000		18	
0.164013	0.000000	0.164071		48.781435	0		200		46080000		18	
0.158533	0.000000	0.158593		49.931992	0		200		46080000		18	
*** Stopping 0 controllers

*** Stopping 2 links
..
*** Stopping 0 switches

*** Stopping 3 hosts
h1 h2 r1 
*** Done
max_queue_size (bytes) = 190000.0
*** Configuring hosts
h1 h2 r1 
[sidecar] Setting congestion control to cubic
[sidecar] tso and gso are OFF
[sidecar] Starting the NGINX/Python webserver on h1...
[sidecar] NOT starting the TCP PEP or sidecar
*** h2 : ('python3 mininet/client.py -n 5000k --http 3 --stdout /dev/null --stderr /home/gina/sidecar/error.log -cc cubic --loss 2 --log-level error -t 2 ',)
head -c 5000k /dev/urandom > /tmp/tmpppx_yd53
Data Size: 5000k
HTTP: --http3
RUST_LOG=error sidecurl --http3 --quiche-cc cubic --data-binary @/tmp/tmpppx_yd53 --insecure https://10.0.1.10:443/ -w "%{time_connect}\t%{time_appconnect}\t%{time_starttransfer}\t\t%{time_total}\t%{exitcode}\t\t%{response_code}\t\t%{size_upload}\t\t%{size_download}\t%{errormsg}\n" --max-time 50 -o /dev/null 2>>/home/gina/sidecar/error.log 
time_connect	time_appconnect	time_starttransfer	time_total	exitcode	response_code	size_upload	size_download	errormsg
0.324456	0.000000	0.324514		10.414381	0		200		5120000		18	
0.158291	0.000000	0.158348		10.579218	0		200		5120000		18	
*** Stopping 0 controllers

*** Stopping 2 links
..
*** Stopping 0 switches

*** Stopping 3 hosts
h1 h2 r1 
*** Done
max_queue_size (bytes) = 190000.0
*** Configuring hosts
h1 h2 r1 
[sidecar] Setting congestion control to cubic
[sidecar] tso and gso are OFF
[sidecar] Starting the NGINX/Python webserver on h1...
[sidecar] NOT starting the TCP PEP or sidecar
*** h2 : ('python3 mininet/client.py -n 12000k --http 3 --stdout /dev/null --stderr /home/gina/sidecar/error.log -cc cubic --loss 2 --log-level error -t 1 ',)
head -c 12000k /dev/urandom > /tmp/tmp3pcwiag3
Data Size: 12000k
HTTP: --http3
RUST_LOG=error sidecurl --http3 --quiche-cc cubic --data-binary @/tmp/tmp3pcwiag3 --insecure https://10.0.1.10:443/ -w "%{time_connect}\t%{time_appconnect}\t%{time_starttransfer}\t\t%{time_total}\t%{exitcode}\t\t%{response_code}\t\t%{size_upload}\t\t%{size_download}\t%{errormsg}\n" --max-time 120 -o /dev/null 2>>/home/gina/sidecar/error.log 
time_connect	time_appconnect	time_starttransfer	time_total	exitcode	response_code	size_upload	size_download	errormsg
0.325251	0.000000	0.325306		25.794443	0		200		12288000		18	
*** Stopping 0 controllers

*** Stopping 2 links
..
*** Stopping 0 switches

*** Stopping 3 hosts
h1 h2 r1 
*** Done
max_queue_size (bytes) = 190000.0
*** Configuring hosts
h1 h2 r1 
[sidecar] Setting congestion control to cubic
[sidecar] tso and gso are OFF
[sidecar] Starting the NGINX/Python webserver on h1...
[sidecar] NOT starting the TCP PEP or sidecar
*** h2 : ('python3 mininet/client.py -n 14000k --http 3 --stdout /dev/null --stderr /home/gina/sidecar/error.log -cc cubic --loss 2 --log-level error -t 1 ',)
head -c 14000k /dev/urandom > /tmp/tmpphyl0edp
Data Size: 14000k
HTTP: --http3
RUST_LOG=error sidecurl --http3 --quiche-cc cubic --data-binary @/tmp/tmpphyl0edp --insecure https://10.0.1.10:443/ -w "%{time_connect}\t%{time_appconnect}\t%{time_starttransfer}\t\t%{time_total}\t%{exitcode}\t\t%{response_code}\t\t%{size_upload}\t\t%{size_download}\t%{errormsg}\n" --max-time 140 -o /dev/null 2>>/home/gina/sidecar/error.log 
time_connect	time_appconnect	time_starttransfer	time_total	exitcode	response_code	size_upload	size_download	errormsg
0.329376	0.000000	0.329434		20.376919	0		200		14336000		18	
*** Stopping 0 controllers

*** Stopping 2 links
..
*** Stopping 0 switches

*** Stopping 3 hosts
h1 h2 r1 
*** Done
max_queue_size (bytes) = 190000.0
*** Configuring hosts
h1 h2 r1 
[sidecar] Setting congestion control to cubic
[sidecar] tso and gso are OFF
[sidecar] Starting the NGINX/Python webserver on h1...
[sidecar] NOT starting the TCP PEP or sidecar
*** h2 : ('python3 mininet/client.py -n 25000k --http 3 --stdout /dev/null --stderr /home/gina/sidecar/error.log -cc cubic --loss 2 --log-level error -t 1 ',)
head -c 25000k /dev/urandom > /tmp/tmphbi5v3d3
Data Size: 25000k
HTTP: --http3
RUST_LOG=error sidecurl --http3 --quiche-cc cubic --data-binary @/tmp/tmphbi5v3d3 --insecure https://10.0.1.10:443/ -w "%{time_connect}\t%{time_appconnect}\t%{time_starttransfer}\t\t%{time_total}\t%{exitcode}\t\t%{response_code}\t\t%{size_upload}\t\t%{size_download}\t%{errormsg}\n" --max-time 250 -o /dev/null 2>>/home/gina/sidecar/error.log 
time_connect	time_appconnect	time_starttransfer	time_total	exitcode	response_code	size_upload	size_download	errormsg
0.321402	0.000000	0.321458		37.768052	0		200		25600000		18	
*** Stopping 0 controllers

*** Stopping 2 links
..
*** Stopping 0 switches

*** Stopping 3 hosts
h1 h2 r1 
*** Done
max_queue_size (bytes) = 190000.0
*** Configuring hosts
h1 h2 r1 
[sidecar] Setting congestion control to cubic
[sidecar] tso and gso are OFF
[sidecar] Starting the NGINX/Python webserver on h1...
[sidecar] NOT starting the TCP PEP or sidecar
*** h2 : ('python3 mininet/client.py -n 35000k --http 3 --stdout /dev/null --stderr /home/gina/sidecar/error.log -cc cubic --loss 2 --log-level error -t 1 ',)
head -c 35000k /dev/urandom > /tmp/tmpjzybado_
Data Size: 35000k
HTTP: --http3
RUST_LOG=error sidecurl --http3 --quiche-cc cubic --data-binary @/tmp/tmpjzybado_ --insecure https://10.0.1.10:443/ -w "%{time_connect}\t%{time_appconnect}\t%{time_starttransfer}\t\t%{time_total}\t%{exitcode}\t\t%{response_code}\t\t%{size_upload}\t\t%{size_download}\t%{errormsg}\n" --max-time 350 -o /dev/null 2>>/home/gina/sidecar/error.log 
time_connect	time_appconnect	time_starttransfer	time_total	exitcode	response_code	size_upload	size_download	errormsg
0.318939	0.000000	0.318999		39.801097	0		200		35840000		18	
*** Stopping 0 controllers

*** Stopping 2 links
..
*** Stopping 0 switches

*** Stopping 3 hosts
h1 h2 r1 
*** Done
max_queue_size (bytes) = 190000.0
*** Configuring hosts
h1 h2 r1 
[sidecar] Setting congestion control to cubic
[sidecar] tso and gso are OFF
[sidecar] Starting the NGINX/Python webserver on h1...
[sidecar] NOT starting the TCP PEP or sidecar
*** h2 : ('python3 mininet/client.py -n 55000k --http 3 --stdout /dev/null --stderr /home/gina/sidecar/error.log -cc cubic --loss 2 --log-level error -t 1 ',)
head -c 55000k /dev/urandom > /tmp/tmp7h8pn2ht
Data Size: 55000k
HTTP: --http3
RUST_LOG=error sidecurl --http3 --quiche-cc cubic --data-binary @/tmp/tmp7h8pn2ht --insecure https://10.0.1.10:443/ -w "%{time_connect}\t%{time_appconnect}\t%{time_starttransfer}\t\t%{time_total}\t%{exitcode}\t\t%{response_code}\t\t%{size_upload}\t\t%{size_download}\t%{errormsg}\n" --max-time 550 -o /dev/null 2>>/home/gina/sidecar/error.log 
time_connect	time_appconnect	time_starttransfer	time_total	exitcode	response_code	size_upload	size_download	errormsg
0.319197	0.000000	0.319253		61.105288	0		200		56320000		18	
*** Stopping 0 controllers

*** Stopping 2 links
..
*** Stopping 0 switches

*** Stopping 3 hosts
h1 h2 r1 
*** Done
max_queue_size (bytes) = 190000.0
*** Configuring hosts
h1 h2 r1 
[sidecar] Setting congestion control to cubic
[sidecar] tso and gso are OFF
[sidecar] Starting the NGINX/Python webserver on h1...
[sidecar] NOT starting the TCP PEP or sidecar
*** h2 : ('python3 mininet/client.py -n 60000k --http 3 --stdout /dev/null --stderr /home/gina/sidecar/error.log -cc cubic --loss 2 --log-level error -t 1 ',)
head -c 60000k /dev/urandom > /tmp/tmpcz45b4ut
Data Size: 60000k
HTTP: --http3
RUST_LOG=error sidecurl --http3 --quiche-cc cubic --data-binary @/tmp/tmpcz45b4ut --insecure https://10.0.1.10:443/ -w "%{time_connect}\t%{time_appconnect}\t%{time_starttransfer}\t\t%{time_total}\t%{exitcode}\t\t%{response_code}\t\t%{size_upload}\t\t%{size_download}\t%{errormsg}\n" --max-time 600 -o /dev/null 2>>/home/gina/sidecar/error.log 
time_connect	time_appconnect	time_starttransfer	time_total	exitcode	response_code	size_upload	size_download	errormsg
0.318909	0.000000	0.318964		63.178238	0		200		61440000		18	
*** Stopping 0 controllers

*** Stopping 2 links
..
*** Stopping 0 switches

*** Stopping 3 hosts
h1 h2 r1 
*** Done
max_queue_size (bytes) = 190000.0
*** Configuring hosts
h1 h2 r1 
[sidecar] Setting congestion control to cubic
[sidecar] tso and gso are OFF
[sidecar] Starting the NGINX/Python webserver on h1...
[sidecar] NOT starting the TCP PEP or sidecar
*** h2 : ('python3 mininet/client.py -n 65000k --http 3 --stdout /dev/null --stderr /home/gina/sidecar/error.log -cc cubic --loss 2 --log-level error -t 1 ',)
head -c 65000k /dev/urandom > /tmp/tmpnkzjqr0m
Data Size: 65000k
HTTP: --http3
RUST_LOG=error sidecurl --http3 --quiche-cc cubic --data-binary @/tmp/tmpnkzjqr0m --insecure https://10.0.1.10:443/ -w "%{time_connect}\t%{time_appconnect}\t%{time_starttransfer}\t\t%{time_total}\t%{exitcode}\t\t%{response_code}\t\t%{size_upload}\t\t%{size_download}\t%{errormsg}\n" --max-time 650 -o /dev/null 2>>/home/gina/sidecar/error.log 
time_connect	time_appconnect	time_starttransfer	time_total	exitcode	response_code	size_upload	size_download	errormsg
0.330145	0.000000	0.330202		70.922115	0		200		66560000		18	
*** Stopping 0 controllers

*** Stopping 2 links
..
*** Stopping 0 switches

*** Stopping 3 hosts
h1 h2 r1 
*** Done
max_queue_size (bytes) = 190000.0
*** Configuring hosts
h1 h2 r1 
[sidecar] Setting congestion control to cubic
[sidecar] tso and gso are OFF
[sidecar] Starting the NGINX/Python webserver on h1...
[sidecar] NOT starting the TCP PEP or sidecar
*** h2 : ('python3 mininet/client.py -n 70000k --http 3 --stdout /dev/null --stderr /home/gina/sidecar/error.log -cc cubic --loss 2 --log-level error -t 1 ',)
head -c 70000k /dev/urandom > /tmp/tmp5g5kdtam
Data Size: 70000k
HTTP: --http3
RUST_LOG=error sidecurl --http3 --quiche-cc cubic --data-binary @/tmp/tmp5g5kdtam --insecure https://10.0.1.10:443/ -w "%{time_connect}\t%{time_appconnect}\t%{time_starttransfer}\t\t%{time_total}\t%{exitcode}\t\t%{response_code}\t\t%{size_upload}\t\t%{size_download}\t%{errormsg}\n" --max-time 700 -o /dev/null 2>>/home/gina/sidecar/error.log 
time_connect	time_appconnect	time_starttransfer	time_total	exitcode	response_code	size_upload	size_download	errormsg
0.318881	0.000000	0.318940		72.978290	0		200		71680000		18	
*** Stopping 0 controllers

*** Stopping 2 links
..
*** Stopping 0 switches

*** Stopping 3 hosts
h1 h2 r1 
*** Done
max_queue_size (bytes) = 190000.0
*** Configuring hosts
h1 h2 r1 
[sidecar] Setting congestion control to cubic
[sidecar] tso and gso are OFF
[sidecar] Starting the NGINX/Python webserver on h1...
[sidecar] NOT starting the TCP PEP or sidecar
*** h2 : ('python3 mininet/client.py -n 75000k --http 3 --stdout /dev/null --stderr /home/gina/sidecar/error.log -cc cubic --loss 2 --log-level error -t 1 ',)
head -c 75000k /dev/urandom > /tmp/tmp2wsulaso
Data Size: 75000k
HTTP: --http3
RUST_LOG=error sidecurl --http3 --quiche-cc cubic --data-binary @/tmp/tmp2wsulaso --insecure https://10.0.1.10:443/ -w "%{time_connect}\t%{time_appconnect}\t%{time_starttransfer}\t\t%{time_total}\t%{exitcode}\t\t%{response_code}\t\t%{size_upload}\t\t%{size_download}\t%{errormsg}\n" --max-time 750 -o /dev/null 2>>/home/gina/sidecar/error.log 
time_connect	time_appconnect	time_starttransfer	time_total	exitcode	response_code	size_upload	size_download	errormsg
0.319157	0.000000	0.319212		79.085376	0		200		76800000		18	
*** Stopping 0 controllers

*** Stopping 2 links
..
*** Stopping 0 switches

*** Stopping 3 hosts
h1 h2 r1 
*** Done
max_queue_size (bytes) = 190000.0
*** Configuring hosts
h1 h2 r1 
[sidecar] Setting congestion control to cubic
[sidecar] tso and gso are OFF
[sidecar] Starting the NGINX/Python webserver on h1...
[sidecar] NOT starting the TCP PEP or sidecar
*** h2 : ('python3 mininet/client.py -n 80000k --http 3 --stdout /dev/null --stderr /home/gina/sidecar/error.log -cc cubic --loss 2 --log-level error -t 1 ',)
head -c 80000k /dev/urandom > /tmp/tmp_wggw7tf
Data Size: 80000k
HTTP: --http3
RUST_LOG=error sidecurl --http3 --quiche-cc cubic --data-binary @/tmp/tmp_wggw7tf --insecure https://10.0.1.10:443/ -w "%{time_connect}\t%{time_appconnect}\t%{time_starttransfer}\t\t%{time_total}\t%{exitcode}\t\t%{response_code}\t\t%{size_upload}\t\t%{size_download}\t%{errormsg}\n" --max-time 800 -o /dev/null 2>>/home/gina/sidecar/error.log 
time_connect	time_appconnect	time_starttransfer	time_total	exitcode	response_code	size_upload	size_download	errormsg
0.330120	0.000000	0.330179		83.113597	0		200		81920000		18	
*** Stopping 0 controllers

*** Stopping 2 links
..
*** Stopping 0 switches

*** Stopping 3 hosts
h1 h2 r1 
*** Done
max_queue_size (bytes) = 190000.0
*** Configuring hosts
h1 h2 r1 
[sidecar] Setting congestion control to cubic
[sidecar] tso and gso are OFF
[sidecar] Starting the NGINX/Python webserver on h1...
[sidecar] NOT starting the TCP PEP or sidecar
*** h2 : ('python3 mininet/client.py -n 85000k --http 3 --stdout /dev/null --stderr /home/gina/sidecar/error.log -cc cubic --loss 2 --log-level error -t 1 ',)
head -c 85000k /dev/urandom > /tmp/tmprhyyhe4x
Data Size: 85000k
HTTP: --http3
RUST_LOG=error sidecurl --http3 --quiche-cc cubic --data-binary @/tmp/tmprhyyhe4x --insecure https://10.0.1.10:443/ -w "%{time_connect}\t%{time_appconnect}\t%{time_starttransfer}\t\t%{time_total}\t%{exitcode}\t\t%{response_code}\t\t%{size_upload}\t\t%{size_download}\t%{errormsg}\n" --max-time 850 -o /dev/null 2>>/home/gina/sidecar/error.log 
time_connect	time_appconnect	time_starttransfer	time_total	exitcode	response_code	size_upload	size_download	errormsg
1.341107	0.000000	1.341166		90.664229	0		200		87040000		18	
*** Stopping 0 controllers

*** Stopping 2 links
..
*** Stopping 0 switches

*** Stopping 3 hosts
h1 h2 r1 
*** Done
max_queue_size (bytes) = 190000.0
*** Configuring hosts
h1 h2 r1 
[sidecar] Setting congestion control to cubic
[sidecar] tso and gso are OFF
[sidecar] Starting the NGINX/Python webserver on h1...
[sidecar] NOT starting the TCP PEP or sidecar
*** h2 : ('python3 mininet/client.py -n 90000k --http 3 --stdout /dev/null --stderr /home/gina/sidecar/error.log -cc cubic --loss 2 --log-level error -t 1 ',)
head -c 90000k /dev/urandom > /tmp/tmpel4qqkb7
Data Size: 90000k
HTTP: --http3
RUST_LOG=error sidecurl --http3 --quiche-cc cubic --data-binary @/tmp/tmpel4qqkb7 --insecure https://10.0.1.10:443/ -w "%{time_connect}\t%{time_appconnect}\t%{time_starttransfer}\t\t%{time_total}\t%{exitcode}\t\t%{response_code}\t\t%{size_upload}\t\t%{size_download}\t%{errormsg}\n" --max-time 900 -o /dev/null 2>>/home/gina/sidecar/error.log 
time_connect	time_appconnect	time_starttransfer	time_total	exitcode	response_code	size_upload	size_download	errormsg
0.330211	0.000000	0.330267		93.578633	0		200		92160000		18	
*** Stopping 0 controllers

*** Stopping 2 links
..
*** Stopping 0 switches

*** Stopping 3 hosts
h1 h2 r1 
*** Done
max_queue_size (bytes) = 190000.0
*** Configuring hosts
h1 h2 r1 
[sidecar] Setting congestion control to cubic
[sidecar] tso and gso are OFF
[sidecar] Starting the NGINX/Python webserver on h1...
[sidecar] NOT starting the TCP PEP or sidecar
*** h2 : ('python3 mininet/client.py -n 95000k --http 3 --stdout /dev/null --stderr /home/gina/sidecar/error.log -cc cubic --loss 2 --log-level error -t 1 ',)
head -c 95000k /dev/urandom > /tmp/tmpk8g1c17w
Data Size: 95000k
HTTP: --http3
RUST_LOG=error sidecurl --http3 --quiche-cc cubic --data-binary @/tmp/tmpk8g1c17w --insecure https://10.0.1.10:443/ -w "%{time_connect}\t%{time_appconnect}\t%{time_starttransfer}\t\t%{time_total}\t%{exitcode}\t\t%{response_code}\t\t%{size_upload}\t\t%{size_download}\t%{errormsg}\n" --max-time 950 -o /dev/null 2>>/home/gina/sidecar/error.log 
time_connect	time_appconnect	time_starttransfer	time_total	exitcode	response_code	size_upload	size_download	errormsg
0.319246	0.000000	0.319304		97.675758	0		200		97280000		18	
*** Stopping 0 controllers

*** Stopping 2 links
..
*** Stopping 0 switches

*** Stopping 3 hosts
h1 h2 r1 
*** Done
max_queue_size (bytes) = 190000.0
*** Configuring hosts
h1 h2 r1 
[sidecar] Setting congestion control to cubic
[sidecar] tso and gso are OFF
[sidecar] Starting the NGINX/Python webserver on h1...
[sidecar] NOT starting the TCP PEP or sidecar
*** h2 : ('python3 mininet/client.py -n 100000k --http 3 --stdout /dev/null --stderr /home/gina/sidecar/error.log -cc cubic --loss 2 --log-level error -t 1 ',)
head -c 100000k /dev/urandom > /tmp/tmp9xf1lmr4
Data Size: 100000k
HTTP: --http3
RUST_LOG=error sidecurl --http3 --quiche-cc cubic --data-binary @/tmp/tmp9xf1lmr4 --insecure https://10.0.1.10:443/ -w "%{time_connect}\t%{time_appconnect}\t%{time_starttransfer}\t\t%{time_total}\t%{exitcode}\t\t%{response_code}\t\t%{size_upload}\t\t%{size_download}\t%{errormsg}\n" --max-time 1000 -o /dev/null 2>>/home/gina/sidecar/error.log 
time_connect	time_appconnect	time_starttransfer	time_total	exitcode	response_code	size_upload	size_download	errormsg
0.318907	0.000000	0.318963		103.096181	0		200		102400000		18	
*** Stopping 0 controllers

*** Stopping 2 links
..
*** Stopping 0 switches

*** Stopping 3 hosts
h1 h2 r1 
*** Done
max_queue_size (bytes) = 190000.0
